astropy/analytic_functions/blackbody.py

```diff
@@ -66,7 +66,7 @@ def blackbody_nu(in_x, temperature):
     # Check if input values are physically possible
     if temp < 0:
         raise ValueError('Invalid temperature {0}'.format(temp))
-    if np.any(freq <= 0):  # pragma: no cover
+    if not np.all(np.isfinite(freq)) or np.any(freq <= 0):  # pragma: no cover
```

Comment:
@embray - do you know what this `pragma: no cover` line is for? This test is in fact covered. Should I remove it?